config/schemas/config.yaml

```diff
@@ -2318,6 +2318,7 @@ properties:
             type: boolean
             default: false
           syncDefaultBuckets:
+            # TODO: if azure and this enabled, warn
```

Comment:
Question: Are these TODOs reminders for yourself or is it something we should fix later?

Reply:
Reminders for myself for things to fix in this PR